packages/jest-runtime/src/index.js

```diff
@@ -496,6 +495,13 @@ class Runtime {
     localModule.parent = mockParentModule;
     localModule.paths = this._resolver.getModulePaths(dirname);
     localModule.require = this._createRequireImplementation(filename, options);
+    localModule.parent = Object.assign({}, localModule.parent, {
+      filename: lastExecutingModulePath,
+      require: this._createRequireImplementation(
```

Comment:
shouldn't this just be `localModule.require`? Why do we create this second one here?